README.md

```diff
@@ -2,102 +2,33 @@
 [![Build Status](https://drone.srcd.host/api/badges/src-d/landing/status.svg)](https://drone.srcd.host/src-d/landing)
 [![Docker Repository on Quay](https://quay.io/repository/srcd/blog/status "Docker Repository on Quay")](https://quay.io/repository/srcd/blog)
 
-This is our [blog](https://blog.sourced.tech) source. It uses fancy technologies, such as:
-- [Hugo](http://gohugo.io/)
-- [Caddy](https://caddyserver.com/)
+This is our [blog](https://blog.sourced.tech) source.
 
-## Available shortcodes
+# Requirements
 
-### Code scroll
+You need to have [Yarn installed](https://yarnpkg.com/en/docs/install) to build the js and css assets. It also lets you locally serve the blog to test and validate it while you are developing.
 
-If your code is too long you can make it have a fixed height and be scrollable.
-
-```
-{{% codescroll height="500" %}}
-\`\`\`js
-my fancy code
-\`\`\`
-{{% /codescroll %}}
-```
-
-### Center
-
-Center the text
-
-```
-{{% center %}}
-some text
-{{% /center %}}
-```
-
-### Caption
-
-Renders an image with caption
-
-```
-{{% caption src="url to image" title="title of image" %}}
-caption of the image, html and md allowed
-{{% /caption %}}
+You can ensure if it is available in your machine running:
+```shell
+yarn --version; # prints your Yarn version
 ```
 
-### YouTube video
+The blog also uses:
+- [Hugo](http://gohugo.io): to build the static html from the markdown content files; it is automatically downloaded and installed at building time.
+- [Caddy](https://caddyserver.com): to serve the static files in production; it is provided by the caddy docker image
 
-Display a YouTube video
+# Build
 
-```
-{{% youtube VIDEO_ID %}}
-```
-
-### Gist
-
-Display a Gist (even an IPython Notebook)
+You need to satisfy the [project requirements](#requirements), and then to run:
 
-```
-{{% gist username gist_id %}}
+```shell
+make build;
 ```
 
-### Grid of 2 elements
+# Contributing
 
-```
-{{% grid %}}
-{{% grid-cell %}}
-SOMETHING LEFT
-{{% /grid-cell %}}
-
-{{% grid-cell %}}
-SOMETHING RIGHT
-{{% /grid-cell %}}
-{{% /grid %}}
-```
-
-### Tweet
-
-Renders a tweet
-
-```
-{{% tweet TWEET_ID %}}
-```
-
-### Anchor
-
-Anchor you can link to with `[foo](#anchor)`:
-
-```
-{{% anchor "anchor" %}}
-```
-
-### codepen
-
-An embedded codepen. Whenever possible, use the sourced account.
-
-```
-{{% codepen slug="aaaaaa" title="My post example" %}}
-```
+If you want to contribute to this project, you will find more info in the [CONTRIBUTING.md](CONTRIBUTING.md)
 
-Available options are:
+# License
 
-- `height` (_optional_, defaults to 573) height of the codepen.
-- `slug` (_required_) the slug of the pen. Usually the last part of its URL
-  (for a sourced pen, the url looks like codepen.io/sourced/pen/slug).
-- `user` (_optional_, defaults to sourced) the user in codepen.
-- `title`(_optional_, defaults to sourced) a title for the codepen.
+Copyright (c) 2015 Máximo Cuadros, see [LICENSE](LICENSE)
```

Comment:
Shouldn't it be source{d}?

Reply:
Since it was defined by @mcuadros  in that way from the very beginning, I didn't want to relicense it in a documentation issue.

I opened an issue https://github.com/src-d/blog/issues/151 to address this new feature request.

Reply:
This was defined by me in my personal repository, witch I deleted, https://github.com/smola/autohugo (but Santiago still has a copy)

Because this project was based on autohugo.

Reply:
Thanks for the confirmation! I'll create a PR with the relicensing once this one is merged :D